chat-plugins/badges.js

```diff
@@ -53,43 +54,70 @@ exports.commands = {
+			let selectedBadge;
 			if (!this.can('ban')) return false;
-			if (parts.length !== 2) return this.errorReply("Correct command: `/badges deleteall, badgeName`");
-			selectedBadge = parts[1].trim();
+			if (!target) return this.errorReply("Correct command: `/badges deleteall badgeName`");
+			selectedBadge = target.trim();
 			if (!Db('badgeData').has(selectedBadge)) return this.errorReply("This badge does not exist, please check /badges list");
 			Db('badgeData').delete(selectedBadge);
 			let badgeUserObject = Db('userBadges').object();
 			let users = Object.keys(badgeUserObject);
 			users.forEach(u => Db('userBadges').set(u, (badgeUserObject[u].filter(b => b !== selectedBadge))));
+
 			this.sendReply("The badge with the name '" + selectedBadge + "' deleted.");
 			this.logModCommand(user.name + " removed the badge '" + selectedBadge + ".");
-			break;
-		default:
-			return this.errorReply("Invalid command. Valid commands are `/badges list`, `/badges info, badgeName`, `/badges set, user, badgeName` and `/badges take, user, badgeName`" +
-			 "`/badges create, name, description, img`.");
-		}
+		},
+		transfer: function (target, room, user) {
+			let parts = target.split(',');
+			if (!this.can('ban')) return false;
+			if (parts.length !== 2) return this.errorReply("Correct command: `/badges transfer userfrom, userto`");
+			let userFrom = toId(parts[0].trim());
+			let userTo = toId(parts[1].trim());
+			let targetUser;
+			targetUser = Users.get(userFrom);
+			if (!targetUser || !targetUser.connected) return this.errorReply("The user is not online.");
+			if (!Db('userBadges').has(userFrom)) return this.errorReply("This user doesn't have any badges.");
+			let alts = Object.keys(targetUser.prevNames);
+			if (!alts.some(a => a === userTo)) return this.errorReply("These two accounts are not alts of the same user.");
+			let userToBadges = Db('userBadges').get(userFrom);
+			Db('userBadges').set(toId(userFrom), []);
+			Db('userBadges').set(toId(userTo), userToBadges);
+
+			this.logModCommand(user.name + " transfered " + userFrom + "'s badges to " + userTo + ".");
+			this.sendReply("All badges were taken from '" + userFrom + "' and given to " + userTo + ".");
+		},
+		'': function (target, room, user) {
+			return this.errorReply("Invalid command. Valid commands are `/badges list`, `/badges info badgeName`, `/badges set user, badgeName`, `/badges take user, badgeName`, " +
+				"`/badges transfer userFrom, userTo`, `/badges create name, description, img`, and `/badges delete badgeName`.");
+		},
 	},
-	badgeshelp: ["Valid commands are `/badges list`, `/badges info, badgeName`, `/badges set, user, badgeName` and `/badges take, user, badgeName`."],
+	badgeshelp: ["Valid commands are `/badges list`, `/badges info badgeName`, `/badges set user, badgeName`, `/badges take user, badgeName`, `/badges transfer userFrom, userTo`, `/badges create name, description, img`, and `/badges delete badgeName`."],
```

Comment:
Something like that. If not anyone can use one of them, specify with "Requires [rank symbols]" at the end.

Also, no ``\n``